src/Mod/Sketcher/Gui/EditModeConstraintCoinManager.cpp

```diff
@@ -126,7 +126,7 @@ void EditModeConstraintCoinManager::updateVirtualSpace()
 
     bool isshownvirtualspace = ViewProviderSketchCoinAttorney::isShownVirtualSpace(viewProvider);
 
-    if(constrlist.size() == vConstrType.size()) {
+    if (constrlist.size() == vConstrType.size()) {

```

Comment:
I am not very happy about changing the format of the whole file by introducing an extra space after each if.

I do not enforce code styling in PRs in contributed code (unless there are severe readability problems). But it is different to just change the styling of a whole file.

Reply:
> I am not very happy about changing the format of the whole file by introducing an extra space after each if.
> 

This is kind of a "damned if I do, damned if I don't" situation :smile: . There was another PR where a reviewer (I believe @donovaly) made me add all the spaces before they merged. It was part design though.

> I do not enforce code styling in PRs in contributed code (unless there are severe readability problems). But it is different to just change the styling of a whole file.

IIRC there were parts of the file where `if (...` was used and parts where `if(...` was, so the commit doesn't change the styling of the whole file as much as make the styling consistent. In any case I intentionally made a separate commit for it, and can drop the commit if you wish.

Reply:
I was referring to https://github.com/FreeCAD/FreeCAD/pull/5155#discussion_r745132975.